<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>622</width>
    <height>450</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Smile Detector</string>
  </property>
  <widget class="QWidget" name="centralWidget">
   <layout class="QGridLayout" name="gridLayout">
    <item row="0" column="0">
     <widget class="QTabWidget" name="tabWidget">
      <property name="currentIndex">
       <number>0</number>
      </property>
      <widget class="QWidget" name="tab">
       <property name="enabled">
        <bool>true</bool>
       </property>
       <attribute name="title">
        <string>Главная</string>
       </attribute>
       <widget class="QLabel" name="label_original">
        <property name="geometry">
         <rect>
          <x>301</x>
          <y>200</y>
          <width>301</width>
          <height>201</height>
         </rect>
        </property>
        <property name="autoFillBackground">
         <bool>true</bool>
        </property>
        <property name="text">
         <string/>
        </property>
       </widget>
       <widget class="QCustomPlot" name="widget" native="true">
        <property name="geometry">
         <rect>
          <x>301</x>
          <y>0</y>
          <width>301</width>
          <height>201</height>
         </rect>
        </property>
       </widget>
       <widget class="QLabel" name="label_faces">
        <property name="geometry">
         <rect>
          <x>0</x>
          <y>0</y>
          <width>301</width>
          <height>201</height>
         </rect>
        </property>
        <property name="autoFillBackground">
         <bool>true</bool>
        </property>
        <property name="text">
         <string/>
        </property>
       </widget>
       <widget class="QPushButton" name="pushButton">
        <property name="geometry">
         <rect>
          <x>150</x>
          <y>300</y>
          <width>121</width>
          <height>41</height>
         </rect>
        </property>
        <property name="text">
         <string>Настройки</string>
        </property>
       </widget>
       <widget class="QPushButton" name="Start">
        <property name="geometry">
         <rect>
          <x>10</x>
          <y>300</y>
          <width>121</width>
          <height>41</height>
         </rect>
        </property>
        <property name="text">
         <string>Старт</string>
        </property>
       </widget>
      </widget>
      <widget class="QWidget" name="tab_2">
       <attribute name="title">
        <string>Tab 2</string>
       </attribute>
      </widget>
     </widget>
    </item>
   </layout>
  </widget>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <customwidgets>
  <customwidget>
   <class>QCustomPlot</class>
   <extends>QWidget</extends>
   <header>qcustomplot.h</header>
   <container>1</container>
  </customwidget>
 </customwidgets>
 <resources/>
 <connections/>
</ui>
